src/redrose-launcher/main.sh

```
@@ -0,0 +1,80 @@
+drive_path=~/.cache/redrose.qcow2
+
+download_url () {
+    echo -e "${GREEN}[!] Downloading : $1${NC}"
+if [ -f "$iso_path" ]; then
+    rm "$iso_path"
+fi
+if [ -f "$drive_path" ]; then
+    rm "$drive_path"
+fi
+  qemu-img create -q -f qcow2 $drive_path 2G
+  wget -q --show-progress -O $iso_path $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)